.github/workflows/go.yml

```
@@ -458,28 +460,29 @@
           GOMEMLIMIT: 6GiB
           GOGC: -1
 
-      - name: Verify Changed files
-        uses: tj-actions/verify-changed-files@v11.1
-        id: verify-changed-files
+      - name: Filter Changed files
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.4)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)